Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -449,6 +444,10 @@ private LanguageServerSettings.PythonSettings GetSettings(Uri scopeUri = null)
             return settings;
 
         }
+        private void OnInterpreterChanged(object sender, EventArgs e) {
+            UpdateInterpreterExcludes();
+            OnSettingsChanged(sender, e);
+        }
```

Comment:
now on interpreter change we update excludes so that ptvs doesnt' send fs events under the interpreter folder